liblangutil/Token.h

```diff
@@ -322,6 +324,22 @@ namespace TokenTraits
 			tok == Token::Default || tok == Token::For || tok == Token::Break || tok == Token::Continue || tok == Token::Leave ||
 			tok == Token::TrueLiteral || tok == Token::FalseLiteral || tok == Token::HexStringLiteral || tok == Token::Hex;
 	}
+	constexpr bool isExperimentalSolidityKeyword(Token tok)
+	{
+		return tok == Token::Assembly || tok == Token::Contract || tok == Token::External || tok == Token::Fallback ||
+			tok == Token::Pragma || tok == Token::Import || tok == Token::As || tok == Token::Function || tok == Token::Let ||
+			tok == Token::Return || tok == Token::Type || tok == Token::Bool || tok == Token::If || tok == Token::Else ||
+			tok == Token::Do || tok == Token::While || tok == Token::For || tok == Token::Continue || tok == Token::Break;
+			// TODO: see isExperimentalSolidityKeyword below
+			// || (tok > Token::NonExperimentalEnd && tok < Token::ExperimentalEnd);
+	}
+	constexpr bool isExperimentalSolidityOnlyKeyword(Token)
+	{
+		// TODO: use token > Token::NonExperimentalEnd && token < Token::ExperimentalEnd
+		// as soon as other experimental tokens are added. For now the comparison generates
+		// a warning from clang because it is always false.
```

Comment:
How annoying :-).

Reply:
Personally, I'd just drop these enum values for now and add them when (if?) we actually need them.

Reply:
We'll need them - clang being weird here is annoying, but it's fine.